nixos/modules/misc/mandoc.nix

```diff
@@ -36,28 +37,35 @@ in {
           configuration options used for the package.
         '';
       };
+
+      extraConfig = lib.mkOption {
```

Comment:
`config` was okay here and even fits better than `extraConfig` as it is the entire config, not just something that gets appended…

Reply:
I'm thinking about implementing something adhere to rfc42. Is that a good idea?

Reply:
That would also work of course! `man.conf(5)` shows that it is very tightly specified, so I would not go the way of making a generic format, but rather try to tightly  map `settings` to the file—don't use `freeformType` since we can't know how new fields in `man.conf` will work in the future and the syntax is not uniform today, i.e. `manpath` and `output` use a different syntax.